reading-list.njk

```diff
@@ -8,11 +8,13 @@ pageHeading: Reading List
   <h3>Season {{ season.number }} - <u>{{ season.title }}</u> by {{ season.author }}</h3>
   <ul>
   {%- for episode in season.episodes -%}
+    {% set ep = episode | getEpisodeEntry(season, collections.episodes) %}
```

Comment:
Can you name `ep` to `releasedEpisode` or something like that?

Reply:
yep